composer.json

```diff
@@ -187,6 +187,18 @@
             "src/Symfony/Component/VarDumper/Resources/functions/dump.php"
         ]
     },
+    "scripts": {
+        "docker-build": "docker build --tag symfony-tests .docker/",
+        "docker-run": "docker run --rm --tty --user $(id --user):$(id --group) --volume $(pwd):/symfony symfony-tests",
+        "docker-composer": "@docker-run env COMPOSER_ROOT_VERSION=6.2.x-dev composer update",
+        "docker-tests": "@docker-run php ./phpunit"
```

Comment:
running tests should be done via `./phpunit src/Symfony/`

Reply:
This runs chunks of tests in parallell instead of one big PHPUnit job with about 40k tests. Thanks!

Reply:
I used this as a default value but now it pass it as a default argument:

```
$ composer docker-tests src/Symfony/Component/Yaml/
docker run […] symfony-tests './phpunit' 'src/Symfony/' 'src/Symfony/Component/Yaml/'
```

So it runs too many tests.